doc/zh-cn/plugins/node-status.md

```diff
@@ -0,0 +1,142 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+
+# 目录
+- [**插件简介**](#插件简介)
+- [**插件属性**](#插件属性)
+- [**插件接口**](#接口插件)
+- [**启用插件**](#启用插件)
+- [**测试插件**](#测试插件)
+- [**禁用插件**](#禁用插件)
+
+
+## 插件简介
+
+`node-status` 是APISIX的请求状态查询插件，返回基本的状态信息。
+
+
+## 插件属性
+
+插件属性为空。
+
+
+## 插件接口
+
+插件增加接口`/apisix/status`，可通过 [interceptors](plugin-interceptors.md)保护该接口。
+
+
+## 启用插件
+
+1. 配置文件apisix/conf/conf.yaml的plugins list中配置node-status
+
+```
+plugins:                          # plugin list
+  - example-plugin
+  - limit-req
+  - node-status
+  - jwt-auth
+  - zipkin
+  ......
+```
+
+启动APISIX之后，即可访问该插件提供的接口，获得基本的状态信息。
+
+2. 在创建route时添加插件node-status
+
+```
+$ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -i -d '
+{
+    "uri": "/route1",
+    "methods": ["PUT", "GET"],
```

Comment:
we can remove this line, it is not important

Reply:
Thanks for your comments, and here you mean remove from line 61 to line 81?

Reply:
only remove `"methods": ["PUT", "GET"],` ^_^

Reply:
OK, I will remove it now~